### PR TITLE
Add relative error measure when (value > 1)

### PR DESCRIPTION
**Pull request by grygielski**
This PR resolves problem with tests failing when compared values are large in magnitude because checking for absolute error between them is often wrong due to floating point calculation precision.

### CODE REVIEW

**Comment by grygielski** [comment]
@luotao1 @bingyanghuang @Sand3r- Please review and share your thoughts on this solution. I used same accuracy threshold for both relative and absolute errors.

File: paddle/fluid/inference/tests/api/tester_helper.h [hunk 1]
```diff
@@ -169,7 +174,12 @@ void CompareResult(const std::vector<PaddleTensor> &outputs,
         float *pdata_ref = ref_out.data<float>(&place, &ref_size);
         EXPECT_EQ(size, ref_size);
         for (size_t j = 0; j < size; ++j) {
-          CHECK_LE(std::abs(pdata_ref[j] - pdata[j]), FLAGS_accuracy);
+          if (std::abs(pdata_ref[j]) > 1) {
+            CHECK_LE(std::abs((pdata_ref[j] - pdata[j]) / pdata_ref[j]),
+                     FLAGS_accuracy);
+          } else {
+            CHECK_LE(std::abs(pdata_ref[j] - pdata[j]), FLAGS_accuracy);
+          }
```
Comment (luotao1) [inline paddle/fluid/inference/tests/api/tester_helper.h]: Could you create a function to do this? It seems duplicated codes.

Comment (grygielski) [reply paddle/fluid/inference/tests/api/tester_helper.h]: Sure, I'll do that

**Review by Sand3r- — APPROVED** [review]
LGTM

**Pull request merged**